src/perf/lib/SecNetPerfMain.cpp

```diff
@@ -87,8 +87,10 @@ PrintHelp(
         "  -cibir:<hex_bytes>          A CIBIR well-known idenfitier.\n"
         "\n"
         "Client: secnetperf -TestName:<Throughput|RPS|HPS> [options]\n"
+#ifndef _KERNEL_MODE
         "Both:\n"
         "  -cpu:<cpu_index>            Specify a processor for raw datapath thread(s) to run on.\n"
```

Comment:
Elsewhere you removed "raw" from the terminology; maybe you want to remove it here too.

Reply:
Sure. Let me know if you see any other places I missed.